python/paddle/fluid/tests/unittests/test_concat_op.py

```diff
@@ -64,5 +64,16 @@ def test_check_grad(self):
         pass
 
 
+class TestConcatOp4(TestConcatOp):
+    def init_test_data(self):
+        self.x0 = np.random.random((2, 3, 4, 5)).astype('float32')
+        self.x1 = np.random.random((2, 3, 4, 5)).astype('float32')
+        self.x2 = np.random.random((0, 3, 4, 5)).astype('float32')
+        self.axis = 0
+
+    def test_check_grad(self):
+        pass
```

Comment:
Why pass check_grad ?

Reply:
Because it will meet fetch error when there is a null grad